youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -5,7 +5,23 @@
 import subprocess
 import time
 import re
-
+import collections
+from string import ascii_letters
+import random
+from ..utils import expand_path
+# from youtube_dl import YoutubeDL
```

Comment:
when I do this import, I can no longer run the youtube_dl module: "can't find '__main__' module"
how to fix?

for now I duplicated the _NUMERIC_FIELDS field